pallene/ast.lua

```diff
@@ -83,7 +83,15 @@ declare_type("Field", {
 -- note: the following functions are why we need `if type(conss) == "table"` in parser.lua
 --
 
--- Return the variables names declared by a given toplevel node
+-- Returns a sequence containing the variable names declared by the specified
+-- toplevel node.
+--
+-- The following nodes will result in a sequence with one element.
+--     ast.Toplevel.Func, ast.Toplevel.Typealias, ast.Toplevel.Typealias,
+--     ast.Toplevel.Record, ast.Toplevel.Import, and ast.Toplevel.Builtin 
+-- The following nodes will result in a sequence with one or more elements.
+--     ast.Toplevel.Var
+-- 
```

Comment:
I don't know if we need this part of the comment. In the places that use ast.toplevel_names the user should not care whether each case is returning one or more names.

Reply:
Fair enough. I'll remove it.